File: sub/domain/src/model/EmailTransInfo.ts
/**
 * @interface EmailTransInfo 전체목록
 */
export interface EmailTransInfo {
  emailTracsceiveDatetime: string; // 이메일송수신시간
  emailTransceiveTypeCd: string; // 이메일송수신타입
  empId: string; // 사번
  opponentEmailDomainAddr: string; // 상대방메일도메인주소
  fileAttachYn: string; // 첨부파일여부
  fileAttachSize: number; // 첨부파일용량
  emailTitle: string; // 이메일제목
  departmentCd: string; // 부서코드
  regDatetime: string; // 등록시간
  inspectionYn: string; // 점검대상여부
  callRuleResult: string; // 룰호출결과
}

File: zero-rule-server-main/domain/src/main/java/server/domain/entity/EmailTransInfo.java
package server.domain.entity;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class EmailTransInfo {
    private String emailTracsceiveDatetime; // 이메일송수신시간
    private String emailTransceiveTypeCd; // 이메일송수신타입
    private String empId; // 사번
    private String opponentEmailDomainAddr; // 상대방메일도메인주소
    private String fileAttachYn; // 첨부파일여부
    private long fileAttachSize; // 첨부파일용량
    private String emailTitle; // 이메일제목
    private String departmentCd; // 부서코드
    private String regDatetime; // 등록시간
    private String inspectionYn; // 점검대상여부
    private String callRuleResult; // 룰호출결과
}
